src/server/pfs/server/testing/server_test.go

```diff
@@ -2726,15 +2726,14 @@ func TestInspectFile3(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, env.PachClient.PutFile(commit2, "foo", strings.NewReader(fileContent2)))
 
-	fileInfo, err = env.PachClient.InspectFile(commit2, "foo")
-	require.NoError(t, err)
-	require.NotNil(t, fileInfo)
+	_, err = env.PachClient.InspectFile(commit2, "foo")
+	// there is a collision between dir foo and file foo
```

Comment:
I would instead change this test so that there isn't a collision (i.e. change either the file or directory path).